src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;

```

Comment:
The field name 'firstSide' could be more descriptive, consider renaming it to 'length'.